apiserver/facades/client/application/application_test.go

```diff
@@ -962,7 +962,9 @@ func (s *applicationSuite) apiv16() *application.APIv16 {
 		APIv17: &application.APIv17{
 			APIv18: &application.APIv18{
 				APIv19: &application.APIv19{
-					APIBase: s.applicationAPI,
+					APIv20: &application.APIv20{
```

Comment:
I wonder if we even should still want to test v16 specifically. I'd be inclined to just test with the latest facade version, unless there's something specific we are testing in v16 for compatibility reasons.

Reply:
these tests actually test the `Destroy()` and `DestroyUnits()` facade methods which are indeed specifically v16. I'm leaving this for now until we completely remove. Anyways none of these tests should be moved to the latest facades version since those methods are for backward compat only.